api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -1352,6 +1347,22 @@ public void purgeObs_shouldDeleteTheGivenObsFromTheDatabase() throws Exception {
 		obsService.purgeObs(obs);
 		
 		Assert.assertNull(obsService.getObs(7));
+		
+		executeDataSet(COMPLEX_OBS_XML);
+		Obs complexObs = obsService.getComplexObs(44, ComplexObsHandler.RAW_VIEW);
+		// obs #44 is coded by the concept complex #8473 pointing to ImageHandler
+		// ImageHandler inherits AbstractHandler which handles complex data files on disk
+		assertNotNull(complexObs.getComplexData());
+		AdministrationService as = Context.getAdministrationService();
+		File complexObsDir = OpenmrsUtil.getDirectoryInApplicationDataDirectory(as
+		        .getGlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_COMPLEX_OBS_DIR));
```

Comment:
Yes, I reacted here to @teleivo 's comments about the using the Generate Test Case Plugin for Eclipse. While I was doing so I also improved the unit test a little (= the deletion of complex data files).
Eventually I wanted to submit 2.0.x's commit on the master branch, but you have merged the former one on master already.

Also this one here would be the new style, but then I had to add my test case in an already existing test method, this was not done on master where my test case has its own method. Which one do you think is the better one?